scripts/_common.sh

```diff
@@ -1,5 +1,124 @@
 #!/bin/bash
 
 #=================================================
-# COMMON VARIABLES AND CUSTOM HELPERS
+# COMMON VARIABLES
 #=================================================
+
+#=================================================
+# EXPERIMENTAL HELPERS
+#=================================================
+
+ynh_maintenance_mode_ON () {
```

Comment:
`ynh_maintenance_mode_ON` do we need to use this helper?

Reply:
Certainly not, as I told you, I just copied the whole code.